src/main.js

```diff
@@ -18,9 +18,9 @@ let mainWindow
 function createWindow () {
 
   // Add the React DevTools (currently has path hard coded).
-  if (fs.existsSync('/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.0_0')) {
+  if (fs.existsSync('/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.1_0')) {
```

Comment:
hardcoded path

Reply:
I was able to get the react developer tooling to automatically install using https://github.com/MarshallOfSound/electron-devtools-installer